src/database/industry_seestar_helper.erl

```diff
@@ -31,16 +32,27 @@ prepare_insert(NameSpace, Table, Schema, Values) ->
 	    end || {Attribute, AttrType} <- Attributes],
     {lists:flatten(Query), Row}.
 
+-spec prepare_select(iolist(), atom(), [term()], iolist()) -> string().
+prepare_select(NameSpace, Table, Schema, Id) ->
```

Comment:
why isn't this called 'prepare_select_by_id' and it calls a 'prepare_select' that accepts a {K,V} (just like the 'prepare_select_secondary_index' has) and then call it? that way you have just one implementation for the query generator and you are reusing code. 
Also - the name 'prepare_select_secondary_index' is so specific...